lib/crypt_keeper/model.rb

```diff
@@ -127,8 +128,10 @@ def encryptor_klass
 
       # Private: Ensure that the encryptor responds to new
       def ensure_valid_encryptor!
-        unless defined?(encryptor_klass) && encryptor_klass.respond_to?(:new)
-          raise "You must specify a valid encryptor `crypt_keeper :encryptor => :aes`"
+        unless defined?(encryptor_klass) && encryptor_klass.respond_to?(:new) &&
+          encryptor_klass.superclass == Provider::Base
```

Comment:
Instead of checking the superclass, can we check that it responds to `dump` and `load`? The class shouldn't matter IMO.